environment.yml

```diff
@@ -3,73 +3,9 @@ channels:
   - conda-forge
 # - defaults
 dependencies:
-# 1/4 req main
-    - python>=3.8,<=3.10
-    - numpy
-    - pandas
-    - pip
-
-    - py-find-1st
-    - aiohttp
-    - SQLAlchemy
-    - python-telegram-bot<20.0.0
-    - arrow
-    - cachetools
-    - requests
-    - urllib3
-    - jsonschema
-    - TA-Lib
-    - tabulate
-    - jinja2
-    - blosc
-    - sdnotify
-    - fastapi
-    - uvicorn
-    - pyjwt
-    - aiofiles
-    - psutil
-    - colorama
-    - questionary
-    - prompt-toolkit
-    - schedule
-    - python-dateutil
-    - joblib
-    - pyarrow
-
-
-    # ============================
-    # 2/4 req dev
-
-    - coveralls
-    - flake8
-    - mypy
-    - pytest
-    - pytest-asyncio
-    - pytest-cov
-    - pytest-mock
-    - isort
-    - nbconvert
-
-    # ============================
-    # 3/4 req hyperopt
-
-    - scipy
-    - scikit-learn
-    - filelock
-    - scikit-optimize
-    - progressbar2
-    # ============================
-    # 4/4 req plot
-
-    - plotly
-    - jupyter
-
-    - pip:
-        - pycoingecko
-        # - py_find_1st
-        - tables
-        - pytest-random-order
-        - ccxt
-        - flake8-tidy-imports
-        - -e .
-    # - python-rapidjso
+  - python>=3.8,<=3.10
+  - pip
+  - libta-lib=0.4.0 # this is required for ta-lib from requirements.txt
+  - pip:
+    - -r requirements.txt
```

Comment:
`requirements.txt` is not the full equivalent to the dependencies listed below.

`libta-lib` is NOT ta-lib as far as i could determine.

---

Now assuming the above is just a differently named ta-lib - then we'll end up with ta-lib installed twice (with potentially conflicting versions).
`requirements.txt` includes ta-lib - which WILL be installed (pip ignores dependencies already installed by conda). 

With that - the whole conda-file has become pointless.

Reply:
Yes, make sense. I've removed the `environment.yml` and updated the documentation as advised.